src/olympia/files/tests/test_utils_.py

```diff
@@ -356,6 +356,25 @@ def test_raise_error_if_no_optional_id_support(self):
             exc.value.message ==
             'GUID is required for Firefox 47 and below.')
 
+    def test_comments_are_allowed(self):
+        json_string = """
+        {
+            // Required
+            "manifest_version": 2,
+            "name": "My Extension",
+            "version": "versionString",
+
+            // Recommended
+            "default_locale": "en",
+            "description": "A plain text description"
+        }
+        """
+        manifest = utils.ManifestJSONExtractor(
+            '/fake_path', json_string).parse()
+
+        assert manifest['is_webextension'] == True
```

Comment:
ah, didn't catch that earlier. comparisons to `None`, `True` and `False` should use the `is` operator instead of `==`, e.g `assert manifest['is_webextension'] is True`.